utils/kubernetes/kubeutil.py

```diff
@@ -80,6 +81,7 @@ def __init__(self, instance=None):
         self.kubelet_host = self.kubelet_api_url.split(':')[1].lstrip('/')
         self.pods_list_url = urljoin(self.kubelet_api_url, KubeUtil.PODS_LIST_PATH)
         self.kube_health_url = urljoin(self.kubelet_api_url, KubeUtil.KUBELET_HEALTH_PATH)
+        self.kube_label_prefix = os.environ.get('KUBERNETES_LABEL_PREFIX') or self.DEFAULT_LABEL_PREFIX
```

Comment:
For consistence, could you please use a configuration option in kubernetes.yaml instead of an environment variable? I suggest `label_to_tag_prefix`. If you need to configure it though an envvar (directly using our image), we could patch the entrypoint.